app/views/static/index.html.erb

```diff
@@ -2,13 +2,15 @@
 <div class='intro-to-tws cover-photo'>
   <div class='content'>
     <div class='content-box'>
-      <h2 class='cover-photo-lede'>
-        Expand your world, one 
-        real-life conversation at a time.
-      </h2>
+      <h1 class='cover-photo-lede'>
```

Comment:
Two things:
![screen shot 2015-05-05 at 11 49 15 pm](https://cloud.githubusercontent.com/assets/380810/7488610/65219a42-f381-11e4-9b1c-0ae5410598a6.png)


1) This is way too narrow on large screens
2) The dark text looks really bad. It's wicked jarring.